src/searchresultsformaction.cpp

```diff
@@ -0,0 +1,45 @@
+#include "searchresultsformaction.h"
+
+namespace newsboat {
+SearchResultsFormAction::SearchResultsFormAction(View* vv,
+	std::string formstr,
+	Cache* cc,
+	FilterContainer& f,
+	ConfigContainer* cfg,
+	RegexManager& r)
+	: ItemListFormAction(vv, formstr, cc, f, cfg, r) {};
+
+const std::vector<KeyMapHintEntry>& SearchResultsFormAction::get_keymap_hint() const
+{
+	static const std::vector<KeyMapHintEntry> hints = {
+		{OP_QUIT, _("Quit")},
+		{OP_OPEN, _("Open")},
+		{OP_PREVSEARCHRESULTS, _("Prev search results")}
+	};
```

Comment:
I think this should also mention:
- the search key, so people understand that they can refine the search;
- the help key, just like the other dialogs do.

Also, I feel somewhat conflicted about contracting words. How about "Previous Results"?